fem/pfespace.hpp

```diff
@@ -335,6 +338,7 @@ class ParFiniteElementSpace : public FiniteElementSpace
    HYPRE_Int GetMyTDofOffset() const;
 
    virtual const Operator *GetProlongationMatrix() const;
+   virtual const Operator *GetLocalProlongationMatrix() const;
```

Comment:
I'm not sure that `GetLocalProlongationMatrix` is a good name. There is a nice symmetry with `GetProlongationMatrix` and `GetRestrictionMatrix`, but this (just like `GetProlongationMatrix`) actually doesn't return a matrix object, and instead returns a specialized matrix-free operator.

It seems like `GetProlongationMatrix` is probably a legacy name from a time when a `HypreParMatrix` was returned, and so it is not changed for backwards compatibility reasons.

But that is not a concern here, so it doesn't seem to me like we should make a function `GetLocalProlongationMatrix` that does not return a matrix.

Reply:
Changing to `GetLocalProlongationOperator`.